conda/hidapi/bld.bat

```diff
@@ -0,0 +1,2 @@
+vcbuild %SRC_DIR%\windows\hidapi.sln
```

Comment:
Does this put the files in the correct directors (``%PREFIX``) for them to appear in the package? See the libusb example. https://github.com/m-labs/conda-recipes/tree/master/conda/libusb